From patchwork Wed Oct 10 16:10:25 2018
Content-Type: text/plain; charset="utf-8"
MIME-Version: 1.0
Content-Transfer-Encoding: 7bit
X-Patchwork-Submitter: Li Qiang <liq3ea@163.com>
X-Patchwork-Id: 10634805
Return-Path: 
 <qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>
Received: from mail.wl.linuxfoundation.org (pdx-wl-mail.web.codeaurora.org
 [172.30.200.125])
	by pdx-korg-patchwork-2.web.codeaurora.org (Postfix) with ESMTP id 7111C17E3
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 16:11:43 +0000 (UTC)
Received: from mail.wl.linuxfoundation.org (localhost [127.0.0.1])
	by mail.wl.linuxfoundation.org (Postfix) with ESMTP id 445882A8F1
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 16:11:43 +0000 (UTC)
Received: by mail.wl.linuxfoundation.org (Postfix, from userid 486)
	id 38B982A900; Wed, 10 Oct 2018 16:11:43 +0000 (UTC)
X-Spam-Checker-Version: SpamAssassin 3.3.1 (2010-03-16) on
	pdx-wl-mail.web.codeaurora.org
X-Spam-Level: 
X-Spam-Status: No, score=-7.7 required=2.0 tests=BAYES_00,DKIM_INVALID,
	DKIM_SIGNED,FREEMAIL_FROM,MAILING_LIST_MULTI,RCVD_IN_DNSWL_HI autolearn=ham
	version=3.3.1
Received: from lists.gnu.org (lists.gnu.org [208.118.235.17])
	(using TLSv1 with cipher AES256-SHA (256/256 bits))
	(No client certificate requested)
	by mail.wl.linuxfoundation.org (Postfix) with ESMTPS id 986A62A901
	for <patchwork-qemu-devel@patchwork.kernel.org>;
 Wed, 10 Oct 2018 16:11:42 +0000 (UTC)
Received: from localhost ([::1]:58111 helo=lists.gnu.org)
	by lists.gnu.org with esmtp (Exim 4.71)
	(envelope-from
 <qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>)
	id 1gAH5J-0005UA-S0
	for patchwork-qemu-devel@patchwork.kernel.org;
 Wed, 10 Oct 2018 12:11:41 -0400
Received: from eggs.gnu.org ([2001:4830:134:3::10]:54693)
	by lists.gnu.org with esmtp (Exim 4.71)
	(envelope-from <liq3ea@163.com>) id 1gAH4O-0004t4-LW
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 12:10:45 -0400
Received: from Debian-exim by eggs.gnu.org with spam-scanned (Exim 4.71)
	(envelope-from <liq3ea@163.com>) id 1gAH4J-0007bZ-KN
	for qemu-devel@nongnu.org; Wed, 10 Oct 2018 12:10:44 -0400
Received: from m12-13.163.com ([220.181.12.13]:48039)
	by eggs.gnu.org with esmtp (Exim 4.71)
	(envelope-from <liq3ea@163.com>)
	id 1gAH4H-0007Ui-8W; Wed, 10 Oct 2018 12:10:39 -0400
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=163.com;
	s=s110527; h=From:Subject:Date:Message-Id; bh=1bBZuWLts8nDxUkTFA
	7h7iaEg/W7z8F7HYmbbOjQW4A=; b=PJG/l3+sjlshUybatQ8A8zzxVFTIlZyEKS
	oHMy/LqphM0p4adcUuXn8giViYc0urMTSW9NsC2sz1YHjV7dZZnLyQwfN94A60gK
	WEGvJJ63jUnjVqoeCHRUawzSu0VCdKAdfJcYxsxYbgTKn02p/fz2FLpaj52GSAAk
	2qEsCEIZk=
Received: from ubuntu.localdomain (unknown [183.159.200.5])
	by smtp9 (Coremail) with SMTP id DcCowABHP1tyJL5bmTH8MQ--.41238S2;
	Thu, 11 Oct 2018 00:10:28 +0800 (CST)
From: Li Qiang <liq3ea@163.com>
To: ehabkost@redhat.com,
	marcel.apfelbaum@gmail.com
Date: Wed, 10 Oct 2018 09:10:25 -0700
Message-Id: <20181010161025.34796-1-liq3ea@163.com>
X-Mailer: git-send-email 2.17.1
X-CM-TRANSID: DcCowABHP1tyJL5bmTH8MQ--.41238S2
X-Coremail-Antispam: 1Uf129KBjvdXoWrtFW3try5ZFy5XrW3Zr4xZwb_yoW3Gwb_Za
	1UAa1rWrW5GayrKw42yw4rAry8Jwn5AF1rAFWxKw4fu3W8Gw1vqFZ7KwnxXr17WrsrGry3
	Aa1xZF1YyFW7KjkaLaAFLSUrUUUUUb8apTn2vfkv8UJUUUU8Yxn0WfASr-VFAUDa7-sFnT
	9fnUUvcSsGvfC2KfnxnUUI43ZEXa7xRifOz3UUUUU==
X-Originating-IP: [183.159.200.5]
X-CM-SenderInfo: 5oltjvrd6rljoofrz/xtbBZhTbbVaDyfXlSgAAsC
X-detected-operating-system: by eggs.gnu.org: GNU/Linux 3.x
X-Received-From: 220.181.12.13
Subject: [Qemu-devel] [PATCH] machine: fix a typo
X-BeenThere: qemu-devel@nongnu.org
X-Mailman-Version: 2.1.21
Precedence: list
List-Id: <qemu-devel.nongnu.org>
List-Unsubscribe: <https://lists.nongnu.org/mailman/options/qemu-devel>,
	<mailto:qemu-devel-request@nongnu.org?subject=unsubscribe>
List-Archive: <http://lists.nongnu.org/archive/html/qemu-devel/>
List-Post: <mailto:qemu-devel@nongnu.org>
List-Help: <mailto:qemu-devel-request@nongnu.org?subject=help>
List-Subscribe: <https://lists.nongnu.org/mailman/listinfo/qemu-devel>,
	<mailto:qemu-devel-request@nongnu.org?subject=subscribe>
Cc: qemu-trivial@nongnu.org, Li Qiang <liq3ea@163.com>, qemu-devel@nongnu.org
Errors-To: 
 qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org
Sender: "Qemu-devel"
	<qemu-devel-bounces+patchwork-qemu-devel=patchwork.kernel.org@nongnu.org>
X-Virus-Scanned: ClamAV using ClamSMTP

Cc: qemu-trivial@nongnu.org
Signed-off-by: Li Qiang <liq3ea@163.com>
Reviewed-by: Stefano Garzarella <stefanogarzarella@gmail.com>
---
 hw/core/machine.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/core/machine.c b/hw/core/machine.c
index 1987557833..da50ad6de7 100644
--- a/hw/core/machine.c
+++ b/hw/core/machine.c
@@ -636,7 +636,7 @@ static void machine_class_init(ObjectClass *oc, void *data)
         machine_get_memory_encryption, machine_set_memory_encryption,
         &error_abort);
     object_class_property_set_description(oc, "memory-encryption",
-        "Set memory encyption object to use", &error_abort);
+        "Set memory encryption object to use", &error_abort);
 }
 
 static void machine_class_base_init(ObjectClass *oc, void *data)
